EvaText/EvaText/Program.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+
+namespace EvaText
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Welcome to Text Adventure University !");
+            Console.WriteLine("You're about to experience TAU, do you have what it takes?");
+
+            string response1 = Console.ReadLine();
+
+            if (response1 == "Yes")
+            {
+                Console.WriteLine("First we must stop by the Student Center.");
+                Console.WriteLine("Here we have to check your admissions status, and your financial status");
+                Console.WriteLine();
+                Console.WriteLine("Are you ready to enter The Student Center?");
+
+                string response2 = Console.ReadLine();
```

Comment:
When variables start having numbers, it's an indicator that the names could be better.  Consider something like `studentCenterResponse` or something similar instead of `response2`